content/en/blog/_posts/2021-04-08-introducing-indexed-jobs.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Introducing Indexed Jobs"
+date: 2021-05-05
+slug: introducing-indexed-jobs
+url: /blog/2021/04/introducing-indexed-jobs
```

Comment:
I recommend omitting this line and letting the generator work out the URL automatically. Also, the month is mismatched against the article date.

Reply:
Removed